numpy/distutils/misc_util.py

```diff
@@ -2295,6 +2295,41 @@ def get_info(name):
     return g.get(name, g.get(name + "_info", {}))
 
 def show():
+    """Print information about various resources (libraries, library directories,
+    include directories, etc.) in the system on which NumPy was built.
+
+    Information is grouped according to classes (``blas_opt_info`` in the example)
+    that specify relevant resources, and it is formatted to be compatible with
+    ``distutils.setup`` keyword arguments.
+
+    See Also
+    --------
+    get_include : Returns the include directory containing NumPy \\*.h header files.
+
+    Notes
+    -----
+    Classes specifying the information to be printed are defined
+    in the ``numpy.distuitls.system_info`` module.
+
+    Information may include:
+    * language: language used to write the libraries (mostly C or f77)
```

Comment:
suggest to use double backticks around each name before the colon, so those are typeset as code.